lib/tasks/role.rake

```diff
@@ -1,11 +1,12 @@
-namespace :"role" do
+namespace :role do
   desc "Retrieve the API key for the given role"
   task :"retrieve-key", [:role_id] => [:environment] do |t, args|
     begin
       creds = Credentials.first!(role_id: args[:role_id])
-      $stderr.puts creds.api_key
+      puts creds.api_key
     rescue Sequel::NoMatchingRow
-      $stderr.puts "Role '#{args[:role_id]}' not found"
+      $stderr.puts "error: #{args[:role_id]} is not a role"
+      puts
```

Comment:
Note for reviewers: this "puts" is to make sure you get n lines of output when you request keys for n roles, no matter whether the roles exist. This is convenient for scripting.

Reply:
It may be the case that the Role does not have any Credentials created yet. In that case, the user's expectation is that the Credentials will be created now.

I would suggest that you first try and find the role (it's an error if it doesn't exist), then use `Role#api_key`.

Reply:
You should put this kind of note in a comment instead of here.

(BTW, I agree with what @kgilpin said. Plus I don't like the wording change: `foo is not a role` suggests to me that `foo` exists although it's not a role, but something else.)